src/runtime_src/xdp/profile/plugin/aie_trace/aie_trace_plugin.cpp

```diff
@@ -137,18 +137,33 @@ namespace xdp {
 
     // **** Core Module Counters ****
     // NOTE: reset events are dependent on actual profile counter reserved
-    coreCounterStartEvents   = {XAIE_EVENT_ACTIVE_CORE,               XAIE_EVENT_ACTIVE_CORE};
-    coreCounterEndEvents     = {XAIE_EVENT_DISABLED_CORE,             XAIE_EVENT_DISABLED_CORE};
-    coreCounterResetEvents   = {XAIE_EVENT_PERF_CNT_0_CORE,           XAIE_EVENT_PERF_CNT_1_CORE,
-                                XAIE_EVENT_PERF_CNT_2_CORE,           XAIE_EVENT_PERF_CNT_3_CORE};
-    coreCounterEventValues   = {1020, 1040400};
+    if (xrt_core::config::get_aie_trace_counter_type() == "es1") {
+      coreCounterStartEvents   = {XAIE_EVENT_ACTIVE_CORE,             XAIE_EVENT_ACTIVE_CORE};
+      coreCounterEndEvents     = {XAIE_EVENT_DISABLED_CORE,           XAIE_EVENT_DISABLED_CORE};
+      coreCounterResetEvents   = {XAIE_EVENT_PERF_CNT_0_CORE,         XAIE_EVENT_PERF_CNT_1_CORE,
+                                  XAIE_EVENT_PERF_CNT_2_CORE,         XAIE_EVENT_PERF_CNT_3_CORE};
+      coreCounterEventValues   = {1020, 1040400};
```

Comment:
You might wish to define what these magic numbers represent.

Reply:
added comment